packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -94,6 +95,13 @@ const PersonalDetailsForm = props => {
         }
     }, [is_tax_residence_popover_open, is_tin_popover_open]);
 
+    const handleSalutationSelection = event => {
+        if (event.target?.type === 'radio') {
+            setSelectedSalutation(event.target?.value);
+            setFieldValue('salutation', event.target?.value);
```

Comment:
If we're setting `salutation` within the formik values, is there a specific reason to have a separate state for radio buttons? Can't we use `values.salutation` from formik for `RadioGroup` selected prop?

Reply:
yes, thanks for noticing. i have used the formik values.